<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>linearcalc</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>RUN</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="comboBox">
          <item>
           <property name="text">
            <string>選擇向量</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_2">
          <property name="currentText">
           <string>選擇矩陣</string>
          </property>
          <item>
           <property name="text">
            <string>選擇矩陣</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_3">
          <property name="maxVisibleItems">
           <number>11</number>
          </property>
          <item>
           <property name="text">
            <string>選擇指令</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>1</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>2</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>6</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>8</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>.</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>+</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>-</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>*</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>/</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>(</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>norm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cross</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>com</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>proj</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>area</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ispara</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>isorth</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>angle</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pn</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>isli</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ob</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rank</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>trans</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ssl</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>det</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>inv</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>adj</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>null</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eigen_vals</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eigen_vecs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>eigen</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pm_val</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pm_vec</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>pm</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ls</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>lu_l</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>lu_u</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>lu</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="4">
         <widget class="QPushButton" name="pushButton_4">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Cl Input</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="pushButton_5">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Cl Data</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="pushButton_3">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Cl Output</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
